script.js

```diff
@@ -0,0 +1,165 @@
+const toDoForm = document.querySelector("form");
+const toDoInput = toDoForm.querySelector(".input-text");
+const toDoButton = toDoForm.querySelector(".input-button");
+
+const TODO_DB = "todo";
+let toDoList = [];
+
+// localStorage 모델: 웹이 로드 될 때 기존 투두리스트를 가져오기 위함
+const localStorageModel = {
+    // localStorage에 저장
+    save: () => {
+        localStorage.setItem(TODO_DB, JSON.stringify(toDoList));
+    },
+
+    // localStorage에서 로드
+    load: () => {
+        const loaded = localStorage.getItem(TODO_DB);
+        if (loaded !== null) {
```

Comment:
javascript에서 if 문은 truthy, falsy를 자동으로 판단해줍니다!

```suggestion
        if (loaded) {
```

혹은 그냥 한 줄로

```suggestion
        if (localStorage.getItem(TODO_DB)) {
```
로 작성하셔도 좋습니다.